pkg/controllers/disruption/consolidation_test.go

```diff
@@ -2050,29 +2050,29 @@ var _ = Describe("Consolidation", func() {
 				Name: "current-on-demand",
```

Comment:
Seems like we're missing consolidation and drift testing for reserved capacity

Reply:
Yeah, I'm planning on expanding the set of disruption and scheduling tests tonight. I definitely plan on including some.